.gitignore

```diff
@@ -32,3 +32,4 @@ Thumbs.dbi
 *.elc
 .\#*
 *.sublime-workspace
+**.idea
```

Comment:
`.idea/` should work fine or maybe `**/.idea`. The current glob pattern might be confusing.